firebase-appdistribution-api/api.txt

```diff
@@ -16,7 +16,10 @@ package com.google.firebase.appdistribution {
   public interface FirebaseAppDistribution {
     method @NonNull public com.google.android.gms.tasks.Task<com.google.firebase.appdistribution.AppDistributionRelease> checkForNewRelease();
     method @NonNull public static com.google.firebase.appdistribution.FirebaseAppDistribution getInstance();
+    method public void cancelFeedbackNotification();
     method public boolean isTesterSignedIn();
+    method public void showFeedbackNotification(@NonNull int, int);
```

Comment:
Any idea why the `int` is annotated with `@NonNull` here (and also on line 27)? It's weird, but probably harmless.